spectral_cube/tests/test_spectral_cube.py

```diff
@@ -34,6 +34,12 @@
 except ImportError:
     bottleneckOK = False
 
+try:
+    import radio_beam
+    radiobeamOK = True
```

Comment:
We normally use ``RADIOBEAM_INSTALLED`` convention (I like installed better than OK)